tests/cdi/cdi_suite_test.go

```diff
@@ -0,0 +1,72 @@
+package cdi_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"kubevirt.io/kubevirt-ansible/tests"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+// template parameters
+const (
+	pvcEPHTTPNOAUTHURL = "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img"
+	pvcName            = "golden-pvc"
+	vmName             = "test-vm"
+	vmAPIVersion       = "kubevirt.io/v1alpha2"
+	rawPVCFilePath     = "tests/manifests/golden-pvc.yml"
+	rawVMFilePath      = "tests/manifests/test-vm.yml"
+)
+
+func TestCDI(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "CDI Suite")
+}
+
+var _ = BeforeSuite(func() {
+	virtCli, err := kubecli.GetKubevirtClient()
```

Comment:
will change to use `CreateNamespace` once https://github.com/kubevirt/kubevirt/pull/1343 merged